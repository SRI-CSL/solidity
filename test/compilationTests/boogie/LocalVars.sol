pragma solidity ^0.4.23;

contract LocalVars {
    uint contractVar;

    function doSomething(uint param) public returns (uint) {
        uint uninitializedVar;
        uninitializedVar = 7;
        uint localVar1 = param;
        localVar1 = localVar1 + 2;
        uint localVar2 = 5;
        localVar2 = localVar1 + param;
        contractVar = localVar1 + localVar2;
        return contractVar;
    }
}