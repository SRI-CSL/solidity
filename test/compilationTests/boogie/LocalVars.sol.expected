Verifying LocalVars::doSomething: OK
No errors found.
